### ISSUE TITLE
1.7.4 breaks apollo-server

### ISSUE THREAD

**Issue by bigman73** [Urigo/merge-graphql-schemas#309]
I updated to 1.7.4 and apollo-server failed to run.
"Error: Unknown type "Query"." 

Rolling back to 1.7.3 solved the problem - and that was the only change. Nothing was different otherwise, code or environment. 
You may argue that this is an apollo-server issue, but the other issue with file loading failing, https://github.com/Urigo/merge-graphql-schemas/issues/308, seems to point to the same root cause.

**Comment by ardatan** [comment]
Could you try with `1.7.5`?

**Comment by bigman73** [comment]
Will do when I get back to work after the holiday is over and let uou know.

**Comment by bigman73** [comment]
UPDATE: v 1.7.5 seems to be working fine

**Comment by liemdo** [comment]
I got a similar problem after upgrading to 1.7.5. My code worked just fine before upgrading

```
const globOptions = {
  ignore: [`${__dirname}/**/{__mock__,tests}/**`]
};
const queryArray = fileLoader(`${__dirname}/**/query.js`, { globOptions });
```

I got error `ReferenceError: describe is not defined` because the `fileLoader` loads files in `/tests`. The `globOptions` doesn't work anymore. Downgrading to 1.7.3 makes it work again.

**Comment by ardatan** [comment]
@liemdo Could you try with this canary version?
`1.7.6-feb3f52.0`

**Comment by ardatan** [comment]
Available in `1.7.6`!
You can pass `ignore` globs in `globOptions`.
